content/docs/community/automated-testing.md

```diff
@@ -9,10 +9,12 @@ All of our Terraform modules have automated tests. We have two sets of checks:
 
 - First set is running github shared workflow [`feature-branch`](https://github.com/cloudposse/github-actions-workflows-terraform-module/blob/main/.github/workflows/feature-branch.yml) 
 which does basic sanity checks: linting, formatting and docs. These checks are cheap to run and do not require any special permissions, so we run them automatically on every commit.
+You also can run this set of checks locally using ```make precommit/terraform``` command on your host.
```

Comment:
```suggestion
You also can and should run this set of checks locally using `make precommit/terraform` command on your host before committing your changes locally and pushing your PR. You should run them because, among other things, it updates the automatically generated documentation, and if you do not do it on your host, your PR will be blocked until you accept the changes via PR suggestions.
```